doc/application/application.rst

```diff
@@ -53,6 +55,15 @@ These contents are:
   kernel configuration option values are used; if no existing values are
   provided, the kernel's default configuration values are used.
 
+* **Device Tree overlays**: Zephyr uses Device Tree to describe the hardware
+  it runs on. Sometimes it is necessary to modify the definitions in the
+  :file:`.dts` files provided in the board folder in order to customize
+  settings for a particular application. The :file:`<board>.overlay` will only
```

Comment:
I'm not sure this belongs here.

Above, it says:

```
An application in its simplest form has the following contents:
```

That would make the reader think these files are part of even the simplest Zephyr applications, but that's not true.

What about adding this information in a new section under "Application Configuration", titled something like "Application Device Tree Overlays", which makes it clear that board.overlay and dts.overlay are optional?

Reply:
Agreed, will do